### ISSUE TITLE
Connect unique Google Sheet ID per form?

### ISSUE THREAD

**Issue by sjcallender** [verbb/formie#1909]
### Question

Is it possible to connect to more than one sheet ID so different forms can export data to different Google Sheets? Currently, it appears only one sheet ID is allowed for the whole plugin.

**Comment by engram-design** [comment]
You select the sheet in the integration settings for the form.

<img width="556" alt="image" src="https://github.com/verbb/formie/assets/1221575/1890fda5-d94e-4f10-a8ee-a33b009bc68c">

If you want to target different _files_ then they indeed need to be separate integrations.

**Comment by sjcallender** [comment]
Yes, sorry, I meant different files. From our view, it would be important to give marketing teams the ability to change files without dev intervention. 

For our context, we moved from Freeform to Formie, in part, because you had the Sheets integration already. We didn't look close enough, but we anticipated more flexibility for our clients. Hoping it's a worthy new feature contender. 🙏🏼

**Comment by engram-design** [comment]
Gotcha. It’s something we can add for sure. Similar to the Webhook integration where there’s a URL set at the integration plugin-level settings, but this can be overridden per form.

I think the only issue may be the token access (the token granted to access one file might not work for another file), but I’ll do some testing.

**Comment by engram-design** [comment]
Updated in [2.2.0](https://github.com/verbb/formie/releases/tag/2.2.0) and [3.1.0](https://github.com/verbb/formie/releases/tag/3.1.0).
